src/utils/mirvanity.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+echo $(basename $0) is not available unless Mir EGL is enabled in Mesa
```

Comment:
I don't think we should be installing a file that does nothing but tell you that it doesn't do anything. Unless it says `please do not press this button again`.

I think the best way of handling this is in `debian/rules`. You should be able to call `dh_install -pmir-utils usr/bin/mirvanity` to manually install the file in the conditional branches under the `override-dh_install:` target.

Reply:
Oh that's probably better indeed.